### PR TITLE
Upstream/93 vor adf

### LINKED ISSUE #93: Preset Increase/Decrease Button vor and adf
Hello,

its possible you can in the programm variable 

short button and long button 

adf 

1 and 10 and 100 for adf inc and dec

vor 1 

1 and 10 inc and dec 

vor 2 

1 and 10 inc and dec

### PR DESCRIPTION
**Pull request by rmroc451**
closes #93

### CODE REVIEW

File: FlightStreamDeck.AddOn/deploy.bat [hunk 1]
```diff
@@ -1,2 +1,3 @@
 taskkill -f -t -im StreamDeck.exe -fi "status eq running"
-XCOPY "." "%appdata%\Elgato\StreamDeck\Plugins\tech.flighttracker.streamdeck.sdPlugin\" /S /E /Y
\ No newline at end of file
+XCOPY "." "%appdata%\Elgato\StreamDeck\Plugins\tech.flighttracker.streamdeck.sdPlugin\" /S /E /Y
+START /d "%ProgramW6432%\Elgato\StreamDeck" StreamDeck.exe
```
Comment (nguyenquyhy) [inline FlightStreamDeck.AddOn/deploy.bat:3]: Do you mind reverting this change for this PR? I'll add it directly in the next commit.

File: FlightStreamDeck.Logics/Actions/PresetChangeAction.cs [hunk 2]
```diff
@@ -137,6 +154,8 @@ private void FlightConnector_AircraftStatusUpdated(object sender, AircraftStatus
             status = e.AircraftStatus;
         }
 
+
```
Comment (nguyenquyhy) [inline FlightStreamDeck.Logics/Actions/PresetChangeAction.cs:157]: Unnecessary new lines?

File: FlightStreamDeck.SimConnectFSX/SimConnectFlightConnector.cs [hunk 3]
```diff
@@ -241,6 +241,11 @@ public void AvMasterToggle(uint state)
             SendCommand(EVENTS.AVIONICS_TOGGLE, state);
         }
 
+        public void GenericSet(TOGGLE_EVENT evt, uint data)
```
Comment (nguyenquyhy) [inline FlightStreamDeck.SimConnectFSX/SimConnectFlightConnector.cs:244]: Can we use existing Trigger function for this? This looks pretty much identical.

**Review by nguyenquyhy — CHANGES_REQUESTED** [review]

File: FlightStreamDeck.AddOn/deploy.bat (hunk 1 above)
Comment (rmroc451) [reply FlightStreamDeck.AddOn/deploy.bat]: updated.

File: FlightStreamDeck.Logics/Actions/PresetChangeAction.cs (hunk 2 above)
Comment (rmroc451) [reply FlightStreamDeck.Logics/Actions/PresetChangeAction.cs]: removed, sorry

File: FlightStreamDeck.SimConnectFSX/SimConnectFlightConnector.cs (hunk 3 above)
Comment (rmroc451) [reply FlightStreamDeck.SimConnectFSX/SimConnectFlightConnector.cs]: I swear the data = 0 was causing an issue when I was testing it earlier, but I just tried it using Trigger, and it works.  So its updated.

**Review by nguyenquyhy — APPROVED** [review]

**Pull request merged**